Runtime/BugSplat.cs

```diff
@@ -206,6 +207,36 @@ string version
 #endif
         }
 
+        /// <summary>
+        /// Constructs and returns a BugSplat object from BugSplatOptions
+        /// </summary>
+        /// <param name="options">collection of options which can be used to configure a BugSplat object </param>
+        /// <param name="application">Your application's name (must match value used to upload symbols)</param>
+        /// <param name="version">Your application's version (must match value used to upload symbols)</param>
+        public static BugSplat CreateFromOptions(BugSplatOptions options, string application, string version)
+        {
+            var bugSplat = new BugSplat(options.Database, application, version);
+
+            bugSplat.Email = options?.Email;
+            bugSplat.Key = options?.Key;
+            bugSplat.User = options?.User;
+            bugSplat.CaptureEditorLog = options.CaptureEditorLog;
+            bugSplat.CapturePlayerLog = options.CapturePlayerLog;
+            bugSplat.CaptureScreenshots = options.CaptureScreenshots;
+
+            if (options.PersistentDataFileAttachmentPaths != null)
+			{
+                foreach (var filePath in options.PersistentDataFileAttachmentPaths)
+                {
+                    var fullFilePath = Path.Combine(Application.persistentDataPath, filePath);
```

Comment:
Possible pain point:
https://stackoverflow.com/questions/53102/why-does-path-combine-not-properly-concatenate-filenames-that-start-with-path-di
Basically, if the second parameter has a leading '/', it will return only the second value. I was running into this issue with `/filename` strings passed in through the BugSplatOptions object, so it might be work either trimming leading '/' or at least leaving a note in the docs.